badge-maker/lib/badge-renderers.js

```diff
@@ -641,7 +639,9 @@ function forTheBadge({
 
   labelWidth += 10 + totalLogoWidth
   if (label.length) {
-    labelWidth += 10 + label.length * 1.5
+    // Add 10 px of padding, plus approximately 1 px of letter spacing per
+    // character.
+    labelWidth += 10 + 2 * label.length
```

Comment:
Should the comment say `approximately 2px of..` or should the calc be `10 + label.length` or perhaps am I just missing something?

Reply:
Yes, you’re totally right. I think these numbers need to be tweaked too. I’ll bring the comment in sync when it stabilizes.